differ/collectionsDiffer.go

```diff
@@ -0,0 +1,46 @@
+package differ
+
+type CollectionsDiffer interface {
+	Diff(incomingCollectionUuids []string, oldCollectionUuids []string) ([]string, map[string]bool)
+}
+
+type defaultCollectionsDiffer struct {
+}
+
+func NewDefaultCollectionsDiffer() *defaultCollectionsDiffer {
+	return &defaultCollectionsDiffer{}
+}
+
+func (dcd *defaultCollectionsDiffer) Diff(incomingCollectionUuids []string, oldCollectionUuids []string) ([]string, map[string]bool) {
```

Comment:
Why not just return the map from this function. The keys from the `isDeleted` are the uuids of the content that was added/deleted.